examples/load_balancer/load_balancer.c

```diff
@@ -143,19 +141,43 @@ usage(const char *progname) {
  */
 static int
 parse_app_args(int argc, char *argv[], const char *progname) {
-        int c;
+        int c, ret;
+        char *client_args, *server_args;
+        char client_ip[32], server_ip[32];
 
         lb->cfg_filename = NULL;
-        lb->client_iface_name = NULL;
-        lb->server_iface_name = NULL;
+        lb->client_port = RTE_MAX_ETHPORTS+1;
```

Comment:
You can just set this to 0 and check if the set port value falls between the valid port range.